packages/backend/src/spec-reader-deployment.spec.ts

```diff
@@ -18,21 +18,27 @@
 
 import { beforeEach, expect, test, vi } from 'vitest';
 import { type Index, SpecReader } from './spec-reader';
-import type { KubeConfig } from '@kubernetes/client-node';
+import { KubeConfig } from '@kubernetes/client-node';
 import index from '../tests/openapi-dump/index.json';
 import appsv1 from '../tests/openapi-dump/openapi/v3/apis/apps/v1.json';
 
 import { type OpenAPIV3 } from 'openapi-types';
 import fetch, { type Response } from 'node-fetch';
+import * as podmanDesktopApi from '@podman-desktop/api';
 
 vi.mock('@kubernetes/client-node');
 vi.mock('node-fetch');
+vi.mock('@podman-desktop/api', () => ({
+  kubernetes: {
+    getKubeconfig: vi.fn(),
+  },
+}));
```

Comment:
I would move this to this file
https://github.com/podman-desktop/extension-kreate/blob/main/__mocks__/%40podman-desktop/api.js

so, no need to copy/paste it

example: https://github.com/minc-org/minc-extension/blob/main/__mocks__/%40podman-desktop/api.js